engines/dotnet/Main.cs

```diff
@@ -159,70 +140,74 @@ static void Main(string[] args)
         //
         // So we do this dance with raw bytes, which is just amazingly
         // inconvenient, because C# doesn't have byte string support.
```

Comment:
Update comment?

Reply:
I didn't want to put words in Andrew's mouth.

Reply:
You can definitely drop the second paragraph now. The code looks much more sensible. :-)